src/wp-admin/options.php

```diff
@@ -169,6 +169,17 @@
 	$allowed_options['writing'][] = 'use_balanceTags';
 }
 
+/**
+ * Filters whether the post-by-email functionality is enabled.
+ *
+ * @since 3.0.0
+ *
+ * @param bool $enabled Whether post-by-email configuration is enabled. Default true.
+ */
+if ( apply_filters( 'enable_post_by_email_configuration', true ) ) {
+	$allowed_options['writing'] = array_merge( $allowed_options['writing'], $mail_options );
+}
+
```

Comment:
What do you think about replacing [line 161](https://github.com/WordPress/wordpress-develop/blob/3e3b4f61309bbb78ac4e11b2f4ca85dda3efd726/src/wp-admin/options.php#L161) with the filter conditional and assigning the `$allowed_options['writing']` values within like the other conditionals do? I think that night read more clearly and consistently, and not have to merge arrays (maybe ever-so-slightly faster?)

Reply:
done

Reply:
done